lib/cli_interface.rb

```diff
@@ -26,20 +24,34 @@ def set_lang(input)
     @text_file = YAML::load(File.open(file))
   end
 
-  def menu_game
-    write("#{get_from_file('menu_type_games')}")
+  def display_menu_type_game(default_checked_type = 1)
```

Comment:
This name could be better, I wasn't entirely sure of what it meant.  Might just be grammar.  e.g. I would say : 'display the game type menu' so my method might be : display_game_type_menu.